lib/plug/conn.ex

```diff
@@ -930,23 +1036,25 @@ defmodule Plug.Conn do
   It returns `{:ok, headers, conn}` with the headers or
   `{:done, conn}` if there are no more parts.
 
-  Once `read_part_headers/2` is invoked, a developer may call
+  Once `read_part_headers/2` is invoked, you may call
   `read_part_body/2` to read the body associated to the headers.
   If `read_part_headers/2` is called instead, the body is automatically
   skipped until the next part headers.
 
   ## Options
 
     * `:length` - sets the maximum number of bytes to read from the body for
-      each chunk, defaults to 64_000 bytes
+      each chunk, defaults to `64_000` bytes
     * `:read_length` - sets the amount of bytes to read at one time from the
-      underlying socket to fill the chunk, defaults to 64_000 bytes
+      underlying socket to fill the chunk, defaults to `64_000` bytes
     * `:read_timeout` - sets the timeout for each socket read, defaults to
-      5_000ms
+      `5_000` milliseconds
 
   """
   @spec read_part_headers(t, Keyword.t()) :: {:ok, headers, t} | {:done, t}
-  def read_part_headers(%Conn{adapter: {adapter, state}} = conn, opts \\ []) do
+  def read_part_headers(conn, opts \\ [])
```

Comment:
Is this change required? There's only a single clause for the `read_part_headers` function